route/route.go

```diff
@@ -155,9 +155,14 @@ func (r *Router) LnS(incomingOrPeer string) {
 	muxxer.HandleFunc("/alive", r.alive).Name("local health")
 	muxxer.HandleFunc("/panic", r.panic).Name("intentional panic")
 	muxxer.HandleFunc("/version", r.version).Name("report version info")
-	muxxer.HandleFunc("/debug/trace/{traceID}", r.debugTrace).Name("get debug information for given trace ID")
-	muxxer.HandleFunc("/debug/rules/{format}/{dataset}", r.getSamplerRules).Name("get formatted sampler rules for given dataset")
-	muxxer.HandleFunc("/debug/allrules/{format}", r.getAllSamplerRules).Name("get formatted sampler rules for all datasets")
+
+	// require a local auth for query usage
```

Comment:
Should we keep the old /debug paths and return an error? Removing these would be a breaking change for anyone who's using them, albeit a feature that's only used to debug.

Reply:
Yeah, they were implemented for the first time in that one; there are no users of this feature.